scripts/ram_bg_color.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+source "$CURRENT_DIR/helpers.sh"
+
+ram_low_bg_color=""
+ram_medium_bg_color=""
+ram_high_bg_color=""
+
+ram_low_default_bg_color="#[bg=green]"
+ram_medium_default_bg_color="#[bg=yellow]"
+ram_high_default_bg_color="#[bg=red]"
+
+get_bg_color_settings() {
+  ram_low_bg_color=$(get_tmux_option "@ram_low_bg_color" "$ram_low_default_bg_color")
+  ram_medium_bg_color=$(get_tmux_option "@ram_medium_bg_color" "$ram_medium_default_bg_color")
+  ram_high_bg_color=$(get_tmux_option "@ram_high_bg_color" "$ram_high_default_bg_color")
+}
+
+print_bg_color() {
+  local ram_percentage=$($CURRENT_DIR/ram_percentage.sh | sed -e 's/%//')
+  local ram_load_status=$(cpu_load_status $ram_percentage)
```

Comment:
`cpu_load_status` should be renamed at some point

Reply:
Not necessarily in this PR

Reply:
Yes probably best to fix later.